tests/analysis_tests/tests/src/expected/Completion.res.txt

```diff
@@ -1000,7 +1000,7 @@ Path Objects.Rec.
 
 Complete src/Completion.res 120:7
 posCursor:[120:7] posNoWhite:[120:6] Found expr:[119:11->123:1]
-posCursor:[120:7] posNoWhite:[120:6] Found expr:[120:5->122:5]
+posCursor:[120:7] posNoWhite:[120:6] Found expr:[120:5->123:0]
```

Comment:
Changes like this are to be expected as the range of a fragment spans from `<>` till end `</>`.